ext/storage_sources/s3_store/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+project(s3_store CXX)    
+
+set(AWSSDK_INCLUDE_DIRS ${INSTALL_DIR}/include)
```

Comment:
`INSTALL_DIR` is an internal detail to the `cmake/third_party/aws_sdk.cmake` and probably shouldn't be used here. 
We should be able to remove this line anyway and any uses of `AWSSDK_INCLUDE_DIRS` in this file, since we will be getting the headers automatically via `INTERFACE_INCLUDE_DIRECTORIES`.